Solutions/Dataminr Pulse/Analytic Rules/DataminrAlertEnrichmentASimDns.yaml

```diff
@@ -0,0 +1,48 @@
+id: e0a69f96-e404-4d05-aceb-621d611a0a62
```

Comment:
This is not a valid use of an alert rule. Creating an incident for each event to generate an automation side effect will strain the system. Also, while you can "auto-close" the incident, the side effects on anything from machine learning to SOC optimization will be significant. You need to find a different method to apply your enrichment.

Reply:
@oshezaf 
Is it fine if we remove the Analytic rule which is creating the Incident, but keep the playbook so that any prospective user can trigger this playbook based on his preferences which can be dependent on any other custom log table data?

Reply:
Yes. Of course.

Reply:
@oshezaf Thanks We will do it.

Reply:
@oshezaf 
As discussed above we have removed the analytics rule which rans the workflow for dataminr alert enrichment.

Cc: @v-prasadboke